eggplant/profiles/views.py

```diff
@@ -103,12 +105,17 @@ def get_initial(self):
 
     def form_valid(self, form):
         user_id = self.request.user.id
+        photo = self.request.FILES.get('photo')
+
         User.objects.filter(id=user_id)\
             .update(first_name=form.cleaned_data['first_name'],
                     last_name=form.cleaned_data['last_name'])
         del form.cleaned_data['first_name']
         del form.cleaned_data['last_name']
+        del form.cleaned_data['photo']
```

Comment:
@jambroo You can't use `.update` because it runs on profile table only. @benjaoming mentioned above use `save`.